src/DPlayer.js

```diff
@@ -33,6 +33,8 @@ class DPlayer {
 
         this.tran = new i18n(this.options.lang).tran;
 
+        this.icons = new Svg(this.options);
```

Comment:
`Svg` seems to represent a single svg instead of a collection of svg, maybe change it to SvgCollection or other name to make it more semantic.